Formula/f/flexiblas.rb

```diff
@@ -21,27 +22,31 @@ class Flexiblas < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "flexiblas-openblas" => :test
   depends_on "gcc" # for gfortran
-  depends_on "openblas"
 
   on_macos do
     depends_on "libomp" => :build
   end
 
+  on_system :linux, macos: :ventura_or_older do
+    depends_on "flexiblas-openblas"
+  end
```

Comment:
I wonder if we should just make Accelerate or Netlib the default everywhere to simplify this. Then depend on `flexiblas-openblas` as needed? Or recommend it in `caveats`.